examples/bevy_wgpu_render.rs

```diff
@@ -167,4 +169,7 @@ fn handle_zoom(keys: Res<ButtonInput<KeyCode>>, mut mats: ResMut<Assets<OctreeVi
 }
 
 #[cfg(not(feature = "bevy_wgpu"))]
-fn main() {} //nothing to do when the feature is not enabled
+fn main() {
+    println!("You probably forgot to enable the bevy_wgpu feature!");
```

Comment:
I find that rather helpful, can make individual commits if preferred though!

Reply:
It's totally fine, I like this addition! It's minor enough to be here